### ISSUE TITLE
Adjust.scenario for multiple groups

### ISSUE THREAD

**Issue by slucey** [NOAA-EDAB/Rpath#21]
Dear Sean,

I hope you are doing well and that this email finds you healthy.

Since I started using Rpath I was wondering if there is a way such that modifications of an Rsim scenario could be streamline as one could do with Rpath model through the read.rpath.params() function that then you could run rpath.stanzas() and later rpath() and with that you have a working Rpath model. With Rsim however it is not possible (or at least I do not know how) to streamline the process of forcing the functions. For instance, when adjusting the scenario for vulnerabilities one needs to do this:

```
Rsim.model <- adjust.scenario(Rsim.model, parameter = "VV",
                              group = "AdHerring", groupto = "AdCod",
                              value = 1)

Rsim.model <- adjust.scenario(Rsim.model, parameter = "VV",
                              group = "JuvSprat", groupto = "AdCod",
                              value = 2.054679E+08)

Rsim.model <- adjust.scenario(Rsim.model, parameter = "VV",
                              group = "AdSprat", groupto = "AdCod",
                              value = 17.36081)
```
 
Rather than something like this:

```
Rsim.model <- adjust.scenario(Rsim.model, parameter = "VV",
                              group = c("AdHerring", "JuvSprat", "AdSprat"),
                              groupto = "AdCod",
                              value = c(1, 2.054679E+08, 17.36081))
```

In the release notes of v0.0.1.2 it reads that now “adjust.scenario can be applied to multiple groups at once rather than one at a time.” The thing though is that from logical R programming one would do what I did above but it is not working. Perhaps because Rpath uses data.table at its core, maybe I need to use a special data.table command to pass to adjust.scenario(). VV is a vector though from what I recall in rpath rather than a matrix as in EwE and the way is populated is special with the outside and primary producers as the first two elements of this vector. It seems that this outside group repeats when checked through `Rsim.model$params$PreyFrom` . Do you have any suggestions as to how pass multiple groups and values at once in the adjust.scenario() function??

Thank you very much in advance as always.

All the absolute best,

Thomas.

**Comment by slucey** [comment]
This is being addressed by pull request #68

**Comment by kaydin** [comment]
Closed following pull request #68 being merged.
